### PR TITLE
Add websocket, transaction and pagination to docs

### PR DESCRIPTION
**Pull request by philippWassibauer**
## Here the example for the first draft so you don't need to load it. 
Screenshot 2024-07-24 at 18 10 02
Screenshot 2024-07-24 at 18 10 07
Screenshot 2024-07-24 at 18 10 28
Screenshot 2024-07-24 at 18 50 09

### CODE REVIEW

File: developer-apis/realtime-openapi.json [hunk 1]
```diff
@@ -162,7 +172,8 @@
                   ],
                   "request_time": "2023-11-07T05:31:56Z",
                   "response_time": "2023-11-07T05:31:56Z",
-                  "wallet_address": "0xd8da6bf26964af9d7eed9e03e53415d37aa96045"
+                  "wallet_address": "0xd8da6bf26964af9d7eed9e03e53415d37aa96045",
+                  "next_offset": "uey8f"
```
Comment (TheEdgeOfRage) [inline developer-apis/realtime-openapi.json:176]: example for an offset for balances: `2Npr8mlkr51-7Z4D5TQV03qpYEUAAAAGIaTZoDegaw3-fWBn7b4QtkeV3uY`

File: developer-apis/transaction-openapi.json [hunk 2]
```diff
@@ -60,7 +58,7 @@
                   "$ref": "#/components/schemas/TransactionsResponse"
                 },
                 "example": {
-                  "next_uri": "https://api.dune.com/beta/transactions/0x7893015832ea33bbe5ca7e4e3045ec2d6a749c8e?block_time=1646055994000000&chain=2&block_number=11506437&index=14&limit=1000",
+                  "next_offset": "uey8f",
```
Comment (TheEdgeOfRage) [inline developer-apis/transaction-openapi.json:61]: Example transactions offset: `AAYddF1M88AAAAAGAAAAAAE2LKQAAAAAAAAAAAAAAAAAAABK`

File: developer-apis/websockets/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+This is crucial for blockchain applications, where timely updates on balances 
+and transactions are essential for user experience and trust. 
+
+Unlike traditional HTTP requests, WebSockets maintain an open connection, 
+reducing latency and overhead associated with repeatedly establishing connections. 
+This efficiency and immediacy make WebSockets an ideal choice for
+ dynamic and interactive blockchain applications.
+
+## What are Websockets and how do they differ from HTTP?
+WebSockets operates as a two-way communication protocol, keeping a persistent connection between a server and a client.
```
Comment (TheEdgeOfRage) [inline developer-apis/websockets/overview.mdx:20]:
```suggestion
WebSockets operate as a two-way communication protocol, keeping a persistent connection between a server and a client. 
```

File: developer-apis/websockets/overview.mdx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## What are Websockets and how do they differ from HTTP?
+WebSockets operates as a two-way communication protocol, keeping a persistent connection between a server and a client. 
+This differs from HTTP, where clients must repeatedly send requests to receive information.
+
+With an active WebSocket connection, clients can receive real-time updates by subscribing to specific network states, 
+like new transactions or blockchain blocks being added.
+
+## Getting Started
+We prepared examples for getting websocket updates in your terminal, python or javascript to help you get started with our Developer Websockets Endpoints.
```
Comment (TheEdgeOfRage) [inline developer-apis/websockets/overview.mdx:27]:
```suggestion
We prepared examples for getting websocket updates in your terminal, python or javascript to help you get started with our Websocket Endpoints.
```

**Review by TheEdgeOfRage — APPROVED** [review]

File: developer-apis/transactions.mdx [hunk 5]
```diff
@@ -21,11 +21,20 @@ Data is typically available within 100 ms after a block was propagated to nodes.
...
+| nonce                           | Nonce of transaction as hexadecimal string                    | string        |
 | transaction_type                | Type of transaction                                           | string        |
-| value                           | Value of transaction                                          | double        |
\ No newline at end of file
+| value                           | Value of transaction as hexadecimal string                    | string        |
+
+
+# Pagination
+
+The response of this endpoint is paginated using a cursor-based pagination. You can use the `limit` parameter to define the maximum page size.
```
[the remark this answers is not in the document]
Comment (markushauge) [reply developer-apis/transactions.mdx:36]: Not resolved 😅

[the remark this answers is not in the document]
Comment (philippWassibauer) [reply developer-apis/transactions.mdx]: now

**Review by markushauge — APPROVED** [review]

**Pull request merged**